core/toolbox/toolbox.js

```diff
@@ -346,6 +346,202 @@ Blockly.Toolbox.prototype.addToolboxItem_ = function(toolboxItem) {
   }
 };
 
+/**
+ * Adds a style on the toolbox. Usually used to change the cursor.
+ * @param {string} style The name of the class to add.
+ * @protected
+ */
+Blockly.Toolbox.prototype.addStyle = function(style) {
+  Blockly.utils.dom.addClass(/** @type {!Element} */ (this.HtmlDiv), style);
+};
+
+/**
+ * Removes a style from the toolbox. Usually used to change the cursor.
+ * @param {string} style The name of the class to remove.
+ * @protected
```

Comment:
If we do keep this, this should be package

Reply:
done